benchmarks/benchmarks/bench_app.py

```diff
@@ -70,8 +70,8 @@ def maxes_of_dots(self, arrays):
 
         Arrays must agree only on the first dimension.
 
-        For numpy it a join benchmark of dot products and max()
-        on a set of arrays.
+        For numpy it a joint benchmark of dot products and
+        max(<array>, axis=<int>).
         """
```

Comment:
I don't find this more enlightening. What does the benchmark actually do?

Reply:
reworded